harvester_e2e_tests/integration/test_vm_functions.py

```diff
@@ -660,8 +661,14 @@ def test_create_schedule_on_maximum(
                     f"VM started but not hosted on expected host: {expected_host}"
                 )
             else:
-                # ???: node's resources released while creating VM, so the VM be hosted on others
-                assert expected_host in [name for name, _ in schedulables], (
+                # ???: node's resources released while creating VM
+                expected_conditions = [
```

Comment:
If `vmi['status']['nodeName']` is the node which VM indeed running currently.
Should the condition be
`vmi['status']['nodeName'] in [name for name, _ in schedulables]`?

Reply:
`schedulables` is containing host that still able to host a VM with maximum resource.

There have 4 conditions:
1. VM be scheduled on the expected host
    a. There still have hosts able to host the VM (`schedulables != []`)
    b. There have **NOT** hosts able to host the VM (`schedulables == []`)
2. VM be **NOT** scheduled on the expected host
    a. There still have hosts able to host the VM (`schedulables != []` and `expected_host in schedulables`)
    b. There have **NOT** hosts able to host the VM (`schedulables == []`)  **<= should not happen**

In here, we adding `1a` as the first `expected_conditions`.